tests/testthat/helper.R

```diff
@@ -6,3 +6,9 @@ local_citation_activate <- function(path, envir = caller_env()) {
   file_move(old, new)
   withr::defer(file_move(new, old), envir = envir)
 }
+
+# Vignette data is cached during as_pkgdown(), so if we create vignettes
+# after local_pkgdown_site() has been run, we need to update the object.
+update_vignettes <- function(pkg) {
```

Comment:
@maelle This doesn't really do much, but it eliminates some head scratching when you see a test that calls `as_pkgdown()` even though we've already created the `pkg` object.